selfdrive/car/docs_definitions.py

```diff
@@ -31,60 +30,83 @@ class Star(Enum):
-class Harness(Enum):
-  nidec = "Honda Nidec"
-  bosch_a = "Honda Bosch A"
-  bosch_b = "Honda Bosch B"
-  toyota = "Toyota"
-  subaru_a = "Subaru A"
-  subaru_b = "Subaru B"
-  fca = "FCA"
-  ram = "Ram"
-  vw = "VW"
-  j533 = "J533"
-  hyundai_a = "Hyundai A"
-  hyundai_b = "Hyundai B"
-  hyundai_c = "Hyundai C"
-  hyundai_d = "Hyundai D"
-  hyundai_e = "Hyundai E"
-  hyundai_f = "Hyundai F"
-  hyundai_g = "Hyundai G"
-  hyundai_h = "Hyundai H"
-  hyundai_i = "Hyundai I"
-  hyundai_j = "Hyundai J"
-  hyundai_k = "Hyundai K"
-  hyundai_l = "Hyundai L"
-  hyundai_m = "Hyundai M"
-  hyundai_n = "Hyundai N"
-  hyundai_o = "Hyundai O"
-  hyundai_p = "Hyundai P"
-  hyundai_q = "Hyundai Q"
-  custom = "Developer"
-  obd_ii = "OBD-II"
-  gm = "GM"
-  nissan_a = "Nissan A"
-  nissan_b = "Nissan B"
-  mazda = "Mazda"
-  ford_q3 = "Ford Q3"
-  ford_q4 = "Ford Q4"
-  none = "None"
-
-
-class HarnessPart(Enum):
+class CarPart(Enum):
+  pass
+
+
+class HarnessConnector(CarPart):
+  nidec = "Honda Nidec connector"
+  bosch_a = "Honda Bosch A connector"
+  bosch_b = "Honda Bosch B connector"
+  toyota = "Toyota connector"
+  subaru_a = "Subaru A connector"
+  subaru_b = "Subaru B connector"
+  fca = "FCA connector"
+  ram = "Ram connector"
+  vw = "VW connector"
+  j533 = "J533 connector"
+  hyundai_a = "Hyundai A connector"
+  hyundai_b = "Hyundai B connector"
+  hyundai_c = "Hyundai C connector"
+  hyundai_d = "Hyundai D connector"
+  hyundai_e = "Hyundai E connector"
+  hyundai_f = "Hyundai F connector"
+  hyundai_g = "Hyundai G connector"
+  hyundai_h = "Hyundai H connector"
+  hyundai_i = "Hyundai I connector"
+  hyundai_j = "Hyundai J connector"
+  hyundai_k = "Hyundai K connector"
+  hyundai_l = "Hyundai L connector"
+  hyundai_m = "Hyundai M connector"
+  hyundai_n = "Hyundai N connector"
+  hyundai_o = "Hyundai O connector"
+  hyundai_p = "Hyundai P connector"
+  hyundai_q = "Hyundai Q connector"
+  custom = "Developer connector"
+  obd_ii = "OBD-II connector"
+  gm = "GM connector"
+  nissan_a = "Nissan A connector"
+  nissan_b = "Nissan B connector"
+  mazda = "Mazda connector"
+  ford_q3 = "Ford Q3 connector"
+  ford_q4 = "Ford Q4 connector"
+
+
+class HarnessAccessory(CarPart):
   harness_box = "harness box"
   comma_power_v2 = "comma power v2"
-  rj45_cable = "RJ45 cable (7 ft)"
+
+
+class Mount(CarPart):
+  mount = "mount"
+  angled_mount = "angled mount"
+
+
+class Cable(CarPart):
+  rj45_cable_7ft = "RJ45 cable (7 ft)"
   long_obdc_cable = "long OBD-C cable"
+  usb_a_2_a_cable = "USB A-A cable"
+  usbc_otg_cable = "USB C OTG cable"
   usbc_coupler = "USB-C coupler"
+  obd_c_cable_1_5ft = "OBD-C cable (1.5 ft)"
 
 
-DEFAULT_HARNESS_PARTS: List[HarnessPart] = [HarnessPart.harness_box, HarnessPart.comma_power_v2, HarnessPart.rj45_cable]
+class Device(CarPart):
+  comma_3 = "comma 3"
+  red_panda = "red panda"
+
+
+DEFAULT_CAR_PARTS: List[CarPart] = [HarnessAccessory.harness_box, HarnessAccessory.comma_power_v2, Cable.rj45_cable_7ft, Mount.mount]
```

Comment:
think every comma three comes with a 1.5ft OBD C cable and is needed (except for some cars like Nissan? I would check)

Reply:
yes they all do. some need longer cables, but that's done with a coupler and long comma two cable